lib/twitter/twitter_user.dart

```diff
@@ -1,16 +1,73 @@
 class TwitterUser {
-  TwitterUser({required this.id, required this.username, required this.name, this.description});
-
-  final String id;
-  final String username;
-  final String name;
-  final String? description;
-
-  factory TwitterUser.fromJson(Map<String, dynamic> json) {
-    return TwitterUser(
-        id: json['id'] as String,
-        username: json['username'] as String,
-        name: json['name'] as String,
-        description: json['description'] as String?);
+  TwitterUser({
+    required this.data,
+    this.includes,
+  });
+
+  late final Data data;
+  late final Includes? includes;
+
+  TwitterUser.fromJson(Map<String, dynamic> json) {
+    data = Data.fromJson(json['data'] as Map<String, dynamic>);
+    includes = json['includes'] != null
+        ? Includes.fromJson(json['includes'] as Map<String, dynamic>)
+        : null;
+  }
+}
+
+class Data {
+  Data({
+    required this.name,
+    required this.id,
+    required this.pinnedTweetId,
+    required this.description,
+    required this.username,
+  });
+
+  late final String name;
+  late final String id;
+  late final String? pinnedTweetId;
+  late final String description;
+  late final String username;
+
+  Data.fromJson(Map<String, dynamic> json) {
+    name = json['name'] as String;
+    id = json['id'] as String;
+    pinnedTweetId = json['pinned_tweet_id'] as String?;
+    description = json['description'] as String;
+    username = json['username'] as String;
+  }
+}
+
+class Includes {
+  Includes({
+    required this.tweets,
+  });
+
+  late final List<Tweets> tweets;
+
+  Includes.fromJson(Map<String, dynamic> json) {
+    tweets = List.from(json['tweets'] as Iterable<dynamic>)
+        .map((e) => Tweets.fromJson(e as Map<String, dynamic>))
+        .toList();
+  }
+}
+
+class Tweets {
+  Tweets({
+    required this.editHistoryTweetIds,
```

Comment:
- Rename to Tweet (classes are always singular)
- we can remove the `editHistory` variable since we have no use for it